sdk/communication/azure-communication-chat/tests/test_chat_thread_client_async.py

```diff
@@ -319,3 +349,32 @@ async def mock_send(*_, **__):
         items.append(item)
 
     assert len(items) == 1
+
+@pytest.mark.asyncio
+async def test_list_read_receipts_with_results_per_page():
+    thread_id = "19:bcaebfba0d314c2aa3e920d38fa3df08@thread.v2"
+    message_id_1 = "1596823919339"
+    message_id_2 = "1596823919340"
+    raised = False
+
+    async def mock_send(*_, **__):
+        return mock_response(status_code=200, json_payload={
+            "value": [
+                {"chatMessageId": message_id_1},
+                {"chatMessageId": message_id_2}
+            ]})
+    chat_thread_client = ChatThreadClient("https://endpoint", credential, thread_id, transport=Mock(send=mock_send))
+
+    read_receipts = None
+    try:
+        read_receipts = chat_thread_client.list_read_receipts(results_per_page=2)
```

Comment:
also a question: If its doable, is it possible to add one more test for testing skip parameter in unit test?

Reply:
This is put in place now 👍